src/models/Offer/Repository.ts

```diff
@@ -44,9 +47,21 @@ export const OfferRepository = {
     status: ApprovalStatus;
   }) =>
     Database.transaction(async transaction => {
+      const isExtension = secretary => secretary === Secretary.extension;
+      const isApproved = chooseStatus => chooseStatus === ApprovalStatus.approved;
+
+      const expirationDate = moment().startOf("day").add(SECRETARY_EXPIRATION_DAYS_SETTING, "days");
```

Comment:
haría end of day (o directamente añadirle esa cantidad de días y que expire a cualquier hora), porque es confuso decirle a alguien que expira en 3 dias cuando es ponele 2.3 días

Reply:
preferible "redondear para arriba"

Reply:
endOfDay si, lo otro no